.travis-requirements.txt

```diff
@@ -1,4 +1,4 @@
 PyGithub
-pyyaml
+pyyaml==5.4.1
```

Comment:
@piyush-J Is #14 an alternative to fixing the version number here?
Or is this needed for some other reason?

Reply:
#14 is the correct alternative, but I think it would be safer to mention version numbers for the libraries to avoid any dependency issues. Please let me know your thoughts.